allensdk/brain_observatory/ecephys/write_nwb/__main__.py

```diff
@@ -14,7 +14,7 @@
 from allensdk.config.manifest import Manifest
 from allensdk.brain_observatory.running_speed import RunningSpeed
 
-from allensdk.brain_observatory.nwb._schemas import InputSchema, OutputSchema
+from ._schemas import InputSchema, OutputSchema
```

Comment:
I thought we decided for explicit as opposed to implicit import?

Reply:
I like the relative better for something like this module's ._schemas file, which is completely tied to the module in whose directory it lives. No strong pref though.